index.html

```diff
@@ -5113,48 +5131,47 @@ <h3 id="playback-AudioBufferSourceNode">
                             </li>
                           </ol>
                         </li>
-                        <li>If <code>loop</code> is true and
-                        <code>bufferTime</code> &gt;=
-                        <code>actualLoopStart</code>,
+                        <li>If <code>loop</code> is true,
                           <ol>
-                            <li>Set <code>enteredLoop</code> to
-                            <code>true</code>.
+                            <li>If <code>offset</code> &lt;
```

Comment:
Maybe worthwhile to add that if enteredLoop is true, we can skip these two items that set enteredLoop to true?  I think once we've set enteredLoop, it's impossible to change it to false.